drivers/sensor/nxp/mcux_lpcmp/Kconfig

```diff
@@ -0,0 +1,18 @@
+# NXP MCUX low power comparator (LPCMP) configuration options
+
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config MCUX_LPCMP
+	bool "NXP LPCMP driver"
+	default y
+	depends on DT_HAS_NXP_LPCMP_ENABLED
+	select PINCTRL
+	help
+	  Enable the MCUX LPCMP driver.
+
+config MCUX_LPCMP_TRIGGER
```

Comment:
I am wondering if the trigger source should be a DT property

Reply:
What's your consideration for thinking it should be a DT property?

Reply:
nevermind, I think I was confusing it with the external trigger sources for round robin mode, which it looks like is not enabled in this PR, that's fine